core/src/miner/mod.rs

```diff
@@ -103,7 +103,7 @@ pub trait MinerService: Send + Sync {
     ) -> Result<(TxHash, u64), Error>;
 
     /// Get a list of all pending transactions in the mem pool.
-    fn ready_transactions(&self, range: Range<u64>) -> PendingVerifiedTransactions;
+    fn ready_transactions<C: EngineInfo>(&self, client: &C, range: Range<u64>) -> PendingVerifiedTransactions;
```

Comment:
How about passing the max body size instead of passing EngineInfo?

Reply:
There will be a concept of `gas` in mold.
So in the future, gas limit parameter will be passed as well.
Do you think passing the two parameters will be better than passing `EngineInfo`?

Reply:
Ok